src/OpenColorIO/GpuShaderUtils.cpp

```diff
@@ -895,27 +921,58 @@ std::string GpuShaderText::sampleTex3D(const std::string& textureName,
 
 void GpuShaderText::declareUniformFloat(const std::string & uniformName)
 {
-    newLine() << (m_lang == GPU_LANGUAGE_MSL_2_0 ? "" : "uniform ") << floatKeyword() << " " << uniformName << ";";
+    std::string uniformDeclString("uniform");
```

Comment:
Very minor but should we add the space here and other declaration methods to avoid having declarations beginning with a white space like we see in the tests?